etcdserver/api/v3rpc/watch.go

```diff
@@ -255,9 +256,10 @@ func (sws *serverWatchStream) recvLoop() error {
 
 				select {
 				case sws.ctrlStream <- wr:
+					break requestHandler
```

Comment:
do we need this goto tag? would a continue work just fine?

Reply:
Ah, thanks for pointing it out. I missed that, fixed.